README.md

```diff
@@ -1,92 +1,113 @@
-    * Boolean fields are prefixed with `is_` or `has_`
-    * Timestamps are appended with `_at`
-    * ID primary keys are prefixed with the name of the table.  For example, the user table's ID column is renamed user_id.
-    * Foreign keys include the table that they refer to. For example, the project table's owner ID column is renamed owner_user_id.
-
-## Installation Instructions
-Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
-
-Include in your `packages.yml`
-
+<p align="center">
+    <a alt="License"
+        href="https://github.com/fivetran/dbt_asana_source/blob/main/LICENSE">
+        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+    <a alt="dbt-core">
+        <img src="https://img.shields.io/badge/dbt_Core™_version->=1.0.0_<2.0.0-orange.svg" /></a>
+    <a alt="Maintained?">
+        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
+    <a alt="PRs">
+        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
+</p>
+
+# Asana Source dbt Package ([Docs](https://fivetran.github.io/dbt_asana_source/))
+# 📣 What does this dbt package do?
+- Materializes [Asana staging tables](https://fivetran.github.io/dbt_asana_source/#!/overview/asana_source/models/?g_v=1&g_e=seeds), which leverage data in the format described by [this ERD](https://fivetran.com/docs/applications/asana#schemainformation). These staging tables clean, test, and prepare your Asana data from [Fivetran's connector](https://fivetran.com/docs/applications/asana) for analysis by doing the following:
+  - Names columns for consistency across all packages and for easier analysis
+  - Removes any rows that are soft-deleted
+  - Adds freshness tests to source data
+  - Adds column-level testing where applicable. For example, all primary keys are tested for uniqueness and non-null values.
+- Generates a comprehensive data dictionary of your Asana data through the [dbt docs site](https://fivetran.github.io/dbt_asana_source/).
+- These tables are designed to work simultaneously with our [Asana transformation package](https://github.com/fivetran/dbt_asana).
+
+# 🎯 How do I use the dbt package?
+## Step 1: Prerequisites
+To use this dbt package, you must have the following:
+
+- At least one Fivetran Asana connector syncing data into your destination.
+- A **BigQuery**, **Snowflake**, **Redshift**, or **PostgreSQL** destination.
+
+## Step 2: Install the package
+Include the following asana_source package version in your `packages.yml` file.
+> TIP: Check [dbt Hub](https://hub.getdbt.com/) for the latest installation instructions, or [read the dbt docs](https://docs.getdbt.com/docs/package-management) for more information on installing packages.
 ```yaml
 packages:
   - package: fivetran/asana_source
     version: [">=0.5.0", "<0.6.0"]
```

Comment:
@fivetran-catfritz FYI we will need to update this to be a new range since these changes are now being combined into a breaking change.
```suggestion
packages:
  - package: fivetran/asana_source
    version: [">=0.6.0", "<0.7.0"]
```